Mathlib/RingTheory/UniqueFactorizationDomain/Defs.lean

```diff
@@ -106,7 +106,7 @@ section Prio
 -- set_option default_priority 100
 
 -- see Note [default priority]
-/-- unique factorization monoids.
+/-- Unique factorization monoids.
```

Comment:
optional, since pre-existing: fold the first line into the second by doing
`Unique factorization monoids are defined as` etc.

Reply:
Done!